db/database.go

```diff
@@ -2336,6 +2346,7 @@ func (db *DatabaseContext) StartOnlineProcesses(ctx context.Context) (returnedEr
 	// If this is an xattr import node, start import feed.  Must be started after the caching DCP feed, as import cfg
 	// subscription relies on the caching feed.
 	if db.autoImport {
+		ctx := db.AddBucketUserLogContext(ctx)
```

Comment:
I'm not sure why this is only being set for import - it seems like it would apply to everything in StartOnlineProcesses?

Reply:
I wanted to be sure that we weren't passing this anywhere we shouldn't be, but I think this is probably true.